light_curves/code/ztf_functions.py

```diff
@@ -56,11 +53,11 @@ def ZTF_get_lightcurve(coords_list, labels_list, nworkers=6, ztf_radius=0.000278
     ztf_df = transform_lightcurves(ztf_df)
 
     # return the light curves as a MultiIndexDFObject
-    indexes, columns = ["objectid", "label", "band", "time"], ["flux", "err"]
+    indexes, columns = ["object_id", "label", "band", "time"], ["flux", "err"]
```

Comment:
If this field name is going to be changed it will need to be updated in all three notebooks under the light_curves/ directory. I think we should avoid this unless there is some strong reason to do it.